app/models/sample/group.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# model to represent sample search group
+class Sample::Group # rubocop:disable Style/ClassAndModuleChildren
```

Comment:
This class should be renamed to `Sample::SearchGroup` for clarity since that's what this file represents